### PR TITLE
fix: resolves issue #332

### PR DESCRIPTION
**Pull request by marcpfuller**
If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-bootstrap/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) N/A
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?) N/A

## Testing Instructions
make test

### CODE REVIEW

File: bootstrap/handlers/metrics.go [hunk 1]
```diff
@@ -50,10 +50,18 @@ func (s *ServiceMetrics) BootstrapHandler(ctx context.Context, wg *sync.WaitGrou
...
-	interval, err := time.ParseDuration(telemetryConfig.Interval)
-	if err != nil {
-		lc.Errorf("Telemetry interval is invalid time duration: %s", err.Error())
-		return false
+	var interval time.Duration
+	var err error
+
+	if telemetryConfig.Interval == "" {
+		lc.Infof("0 specified for metrics reporting interval. Setting to max duration to effectively disable reporting.")
+		interval = math.MaxInt64
```
Comment (lenny-intel) [inline bootstrap/handlers/metrics.go:58]:
```suggestion
   telemetryConfig.Interval = "0s"
```

File: bootstrap/handlers/metrics.go [hunk 2]
```diff
@@ -50,10 +50,18 @@ func (s *ServiceMetrics) BootstrapHandler(ctx context.Context, wg *sync.WaitGrou
...
+	} else {
```
Comment (lenny-intel) [inline bootstrap/handlers/metrics.go:59]:
```suggestion

```

File: bootstrap/handlers/metrics.go [hunk 3]
```diff
@@ -50,10 +50,18 @@ func (s *ServiceMetrics) BootstrapHandler(ctx context.Context, wg *sync.WaitGrou
...
+		interval, err = time.ParseDuration(telemetryConfig.Interval)
```
Comment (lenny-intel) [inline bootstrap/handlers/metrics.go:60]:
```suggestion
		interval, err := time.ParseDuration(telemetryConfig.Interval)
```

File: bootstrap/handlers/metrics.go [hunk 4]
```diff
@@ -50,10 +50,18 @@ func (s *ServiceMetrics) BootstrapHandler(ctx context.Context, wg *sync.WaitGrou
...
+		if err != nil {
+			lc.Errorf("Telemetry interval is invalid time duration: %s", err.Error())
+			return false
+		}
 	}
```
Comment (lenny-intel) [inline bootstrap/handlers/metrics.go:65]:
```suggestion
	
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: bootstrap/handlers/metrics.go (hunk 4 above)
Comment (marcpfuller) [reply bootstrap/handlers/metrics.go]: I have made and  amended the changes suggested

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**